packages/block-library/src/post-author/index.php

```diff
@@ -0,0 +1,32 @@
+<?php
+/**
+ * Server-side rendering of the `core/post-author` block.
+ *
+ * @package WordPress
+ */
+
+/**
+ * Renders the `core/post-author` block on the server.
+ *
+ * @return string Returns the filtered post author for the current post wrapped inside "h6" tags.
+ */
+function render_block_core_post_author() {
+	$post = gutenberg_get_post_from_context();
+	if ( ! $post ) {
+		return '';
+	}
+	return '<h6> By ' . get_the_author( $post ) . '</h6>';
```

Comment:
We have to add a i18n function  `__( 'By ' ),` to make this string translatable.

Reply:
Done 👍